Util/src/main/java/io/deephaven/util/locks/SafeCloseableUnlock.java

```diff
@@ -0,0 +1,19 @@
+package io.deephaven.util.locks;
+
+import io.deephaven.util.SafeCloseable;
+
+import java.util.Objects;
+import java.util.concurrent.locks.Lock;
+
+final class SafeCloseableUnlock implements SafeCloseable {
```

Comment:
Seems like a lot of boilerplate for something that could be a method reference to `this::unlock`.

Reply:
Never mind, I see you did this.